src/window.c

```diff
@@ -157,6 +157,37 @@ log_frame_layout(frame_T *frame)
 }
 #endif
 
+/*
+ * Check if the current window is allowed to move to a different buffer.
+ * If the window has 'winfixbuf', this function will return FALSE.
+ */
+int check_can_set_curbuf_disabled(void)
+{
+    if (curwin->w_p_wfb)
+    {
+	semsg("%s", e_winfixbuf_cannot_go_to_buffer);
+	return FALSE;
```

Comment:
and here please

Reply:
I see what happened here, I forgot to add `check_can_set_curbuf_disabled` which is also why the coverage says that it doesn't run. Will add!

Reply:
This is done

Reply:
okay thanks.